Sources/BraintreeApplePay/BTApplePayClient.swift

```diff
@@ -69,6 +69,19 @@ import BraintreeCore
         }
     }
 
+    /// Checks if the Apple Pay merchant enabled payment networks are supported on the user's device.
+    /// - Returns: A `Bool`
+    public func isApplePaySupported() async -> Bool {
```

Comment:
Wondering if we also want to expose a completion handler vs just the async method. Since this module exposes both elsewhere. What do you think?